tools/ola_trigger/contrib/hue_trigger.conf

```diff
@@ -0,0 +1,47 @@
+# OLA Trigger config to control Philips Hue Lux lights
+# Copyright (C) 2015 Johan Nilsson. www.gobo.ws
+# Thanks to Simon Newton and Sean Sill
+# Resources used from developers.meethue.com
+
+
+
+# DMX protocol
+
+###############################################################################
+
+# Channel 1 lamp on/off. Value 0-140 lamp off. 141-255 lamp on
```

Comment:
Is there some reason the split is 140, rather than 127 in the middle of the range?